dialog-polyfill.js

```diff
@@ -483,4 +483,12 @@
   window['dialogPolyfill'] = dialogPolyfill;
   dialogPolyfill['forceRegisterDialog'] = dialogPolyfill.forceRegisterDialog;
   dialogPolyfill['registerDialog'] = dialogPolyfill.registerDialog;
+
+  if ( typeof module === 'object' && typeof module.exports === 'object' ) {
```

Comment:
I don't know the ins and outs of Closure (I've only ever used Uglify), but I figured that people using Closure probably aren't using CommonJS anyway. :wink: